src/condor_utils/submit_utils.cpp

```diff
@@ -7113,6 +7231,28 @@ int SubmitHash::process_input_file_list(StringList * input_list, long long * acc
 	return 0;
 }
 
+SubmitHash::ContainerImageType 
+SubmitHash::image_type_from_string(const std::string &image) const {
+	if (starts_with(image, "docker:")) {
```

Comment:
is case sensitive test correct here?

Reply:
yes, the docker prefix must be lower case